File: frontend/src/Pages/Announcement.js
import announcements from '../sample_data/announcements.json'

function Announcement() {
    const content = announcements.map((announcement, index) => {
        return (
            <div>
                <h3>{announcement.title}</h3>
                <div>{announcement.content}</div>
            </div>
        );

    });
    return (
        <div>
            <h1>Announcement</h1>
            {content}
        </div>
    );
}

export default Announcement;

File: backend/api/src/main/java/com/rucsrate/api/model/Announcement.java
package com.rucsrate.api.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document("Announcements")
public class Announcement {

    @Id
    private String id;

    private String Title;
    private String Content;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        this.Title = title;
    }

    public String getContent() {
        return Content;
    }

    public void setContent(String content) {
        this.Content = content;
    }
}
